build/depends.py

```diff
@@ -511,6 +511,9 @@ def configure(self, build, conf, env=None):
             # Try using system lib
             if conf.CheckForPKG('soundtouch', '2.0.0'):
                 # System Lib found
+                if not conf.CheckLib(['soundtouch', 'libsoundtouch', 'libSoundTouch']):
```

Comment:
I think CheckLib also adds the library to the linker command line.
Please verify that we have it listed only once.